test/test_distributions.py

```diff
@@ -307,6 +307,185 @@ def is_all_nan(tensor):
     ]),
 ]
 
+BAD_EXAMPLES = [
+    Example(Bernoulli, [
+        {'probs': Variable(torch.Tensor([1.1, 0.2, 0.4]), requires_grad=True)},
```

Comment:
I believe this should be `variable([1.1, 0.2, 0.4])`, we don't use `Variable` anymore.